src/pint/models/parameter.py

```diff
@@ -1253,9 +1253,9 @@ class prefixParameter:
     F22. Appropriate units will be inferred,
 
     To create a prefix parameter with the same prefix but different index, just
-    use the ``.new_param`` method. It will return a new prefix parameter with the
+    use the :meth:`pint.parameter.prefixParameter.new_param` method. It will return a new prefix parameter with the
     same setup but the index. Some parameters' unit and description will
-    be changed once the index has been changed. In order to get the right units
+    be changed once the index has been changed. The new parameter will not inherit the ``frozen`` status of its parent by default.  In order to get the right units
     and description, ``.unitTplt`` and ``.descriptionTplt`` should be provided. If
```

Comment:
Just noticed this, but I think the docs are outdated here.  Should be `unit_template` and `description_template`